src/app/components/connect-applications-wizard/connect-applications-wizard.html

```diff
@@ -0,0 +1,35 @@
+<div class="connect-apps-panel">
+  <!-- Header -->
+  <div class="connect-apps-wrap">
+    <div class="row">
+      <div class="col-md-12">
+        <table class="connect-apps-table table table-bordered">
```

Comment:
I used it as displaying trick, for the distribution of the element. There are not boxes or anything alike displayed though.